code/modules/research/designs/engineering.dm

```diff
@@ -31,6 +31,14 @@
 	build_path = /obj/item/weapon/tool/crowbar/power
 	sort_string = "NAAAC"
 
+/datum/design/item/tool/rpd
+	name = "Rapid Piping Device (RPD)"
+	desc = "A device used to rapidly pipe things."
+	id = "RPD"
+	req_tech = list(TECH_ENGINEERING = 3, TECH_MATERIAL = 2, TECH_BLUESPACE = 2)
+	materials = list(DEFAULT_WALL_MATERIAL = 300, "silver" = 100, "diamond" = 500)
+	build_path = /obj/item/weapon/pipe_dispenser
+	sort_string = "NAAAB"
```

Comment:
Add //YW addition and //YW ends or put this datum in engineering_yw.dm 😄

Reply:
;~; fiiiiine. It’ll be like a week before I can. Am busy.

Reply:
I can fork your code and push it to a branch on this rep and then add the comments, then you don't have to worry about it.

Reply:
or to make it easier just merge it and then i or someone can add the comments